tests/test_flake8_trio.py

```diff
@@ -186,17 +186,24 @@ def find_magic_markers(
 # This could be optimized not to reopen+reread+reparse the same file over and over
 # when testing the same file
 @pytest.mark.parametrize(("test", "path"), test_files)
-@pytest.mark.parametrize("autofix", [False, True])
-@pytest.mark.parametrize("anyio", [False, True])
+@pytest.mark.parametrize("autofix_str", ["noautofix", "autofix"])
+@pytest.mark.parametrize("library", ["trio", "anyio"])
 def test_eval(
-    test: str, path: Path, autofix: bool, anyio: bool, generate_autofix: bool
+    test: str, path: Path, autofix_str: str, library: str, generate_autofix: bool
 ):
+    # pytest prints parametrized values, but not the key, so to make the summary info
+    # readable we make the parameters strings and here turn them into bools
+    autofix = autofix_str == "autofix"
+    anyio = library == "anyio"
```

Comment:
There's also the `ids=` argument which can customize this, passing either a list of strings or an arg->str function.

Reply:
ooh, much better!